src/__tests__/help.ts

```diff
@@ -0,0 +1,24 @@
+import util from 'util';
+
+const exec = util.promisify(require('child_process').exec);
+
+test('npx unimported --help', async () => {
```

Comment:
I'm worried about this test.  I feel like it's just testing `yargs` functionality and will break anytime they change the output format. 

I guess it tests that the cli args are wired up correctly, which has some value... Maybe worth the cost of updating in the rare case `yargs` changes the help output?

Reply:
Agree. If it gets annoying over time, we could always refactor to string matching. Not sure if it's worth doing at this moment. What do you think?

```js
expect(stdout).toContain('--init');
expect(stdout).toContain('dump default settings to .unimportedrc.json');

expect(stdout).toContain('--flow');
expect(stdout).toContain('indicates if your code is annotated with flow types');

expect(stdout).toContain('--update');
expect(stdout).toContain('update the ignore-lists stored in .unimportedrc.json');
```

Reply:
Happy either way. You're likely to be the one that gets bitten by it so I'll let you decide.